backend/json.go

```diff
@@ -62,6 +62,10 @@ func writeDataResponseJSON(dr *DataResponse, stream *jsoniter.Stream) {
 		if !status.IsValid() {
 			status = statusFromError(dr.Error)
 		}
+
+		stream.WriteMore()
+		stream.WriteObjectField("source")
```

Comment:
Nit. Should probably be `errorSource` based on my other comment.

Reply:
I was using "source" here to align with [what is in grafana](https://github.com/grafana/grafana/blob/main/pkg/plugins/backendplugin/instrumentation/instrumentation.go#L46).

Reply:
Oh I see. That label is actually used to denote how a plugin is hosted/run, see values here https://github.com/grafana/grafana/blob/1ce603b9e93153c218ac3e14606d802f305deb6e/pkg/plugins/backendplugin/ifaces.go#L31-L36. We might want to rename this to remove confusion :smile: For HTTP request metrics we use `status_source`, see https://github.com/grafana/grafana/blob/1ce603b9e93153c218ac3e14606d802f305deb6e/pkg/middleware/request_metrics.go#L41 to denote where the HTTP status is originating from.